VTUL/dlp-access
# Add ability to edit podcast episodes

**Add ability to edit podcast episodes. Make sure that all fields for create are also available for edit**
* * *

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2604) (:star:)

* Other Relevant Links (Meeting note, project page, related pull requests, etc.)

# What does this Pull Request do? (:star:)
Add ability to edit podcast episodes. Make sure that all fields for create are also available for edit.

# What's the changes? (:star:)
* Refactored IdentifierForm.js to work for Podcast form
* Loads record details if identifier is submitted
* Added "required" to essential fields

# How should this be tested?
* visit `/siteAdmin`
* click "New / Update Podcast Episode"
* click "Create Podcast Episode"
* fill in all required fields
* click "Submit Podcast Episode"
* check that record was created successfully (take note of identifier)

* visit `/siteAdmin`
* click "New / Update Podcast Episode"
* enter identifier that was created in previous section
* click "Confirm"
* change the value of a field that was created in previous section
* click "Submit Podcast Episode"
* check that the value was successfully changed

# Additional Notes:
* branch: `LIBTD-2604`

# Interested parties
@yinlinchen 

(:star:) Required fields
